app/models/comment.rb

```diff
@@ -601,7 +601,7 @@ def self.recent_threads(user)
 
     thread_ids = Comment
       .where(user: user)
-      .group(:thread_id)
+      .select(:thread_id)
```

Comment:
One more benefit on this update is that we are only requiring the thread_id attribute instead of the whole model's attributes

Reply:
The `pluck` a few lines down already does this.